dispatches/models/fossil_case/concrete_tube/test_heat_exchanger_tube.py

```diff
@@ -0,0 +1,81 @@
+##############################################################################
+# Institute for the Design of Advanced Energy Systems Process Systems
+# Engineering Framework (IDAES PSE Framework) Copyright (c) 2018-2020, by the
+# software owners: The Regents of the University of California, through
+# Lawrence Berkeley National Laboratory,  National Technology & Engineering
+# Solutions of Sandia, LLC, Carnegie Mellon University, West Virginia
+# University Research Corporation, et al. All rights reserved.
+#
+# Please see the files COPYRIGHT.txt and LICENSE.txt for full copyright and
+# license information, respectively. Both files are also available online
+# at the URL "https://github.com/IDAES/idaes-pse".
+##############################################################################
+"""
+Tests for ConcreteTubeSide model.
+
+Author: Konica Mulani, Jaffer Ghouse
+"""
+
+import pytest
+from pyomo.environ import (ConcreteModel, TerminationCondition,
+                           SolverStatus, value, units as pyunits)
+
+from idaes.core import FlowsheetBlock
+from heat_exchanger_tube import ConcreteTubeSide as HX1D
+from idaes.generic_models.unit_models.heat_exchanger \
+    import HeatExchangerFlowPattern
+
+from idaes.generic_models.properties import iapws95
+
+from idaes.core.util.model_statistics import degrees_of_freedom
+
+from idaes.core.util.testing import get_default_solver
+
+
+def test_build():
```

Comment:
This is beyond this PR, but we really should set up some pytest marks for DISPATCHES.